packages/logging-bunyan/test/index.js

```diff
@@ -203,10 +198,47 @@ describe('logging-bunyan', function() {
         done();
       };
 
-      loggingBunyan.write(record);
+      loggingBunyan._formatEntry(record);
     });
   });
 
+  describe('_write', function() {
+    var STACKDRIVER_LEVEL = 'info';
+
+    beforeEach(function() {
+      fakeLogInstance.entry = function() {};
+      loggingBunyan.log_[STACKDRIVER_LEVEL] = function() {};
+    });
+
+    it('should format the record', function(done) {
+      loggingBunyan._formatEntry = function(record) {
+        assert.strictEqual(record, RECORD);
+        done();
+      };
+
+      loggingBunyan._write(RECORD, '', assert.ifError);
+    });
+
+    it('should write to the correct log', function(done) {
+      var customLevel = 'custom-level';
+      var entry = {};
+
+      loggingBunyan.log_.entry = function() {
+        return entry;
+      };
+
+      LoggingBunyan.BUNYAN_TO_STACKDRIVER[RECORD.level] = customLevel;
+
+      loggingBunyan.log_[customLevel] = function(entry_) {
+        assert.strictEqual(entry_, entry);
+        done();
+      };
+
+      loggingBunyan._write(RECORD, '', assert.ifError);
```

Comment:
Pass in `done` in place of `assert.ifError`, then execute it from the `loggingBunyan.log_[customLevel]` override. This will test that the correct callback is being passed to the log function.